process_manager/templates/process_manager/index.html

```diff
@@ -48,7 +48,17 @@
                 type="button"
                 class="btn btn-info"
                 _="on load hide me on click hide me show #message-panel">Show Messages</button>
-        <div hx-get="{% url 'process_manager:process_table' %}"
+        <p>&nbsp;</p>
+        <input class="form-control join-item"
+               type="search"
+               name="search"
+               placeholder="Search..."
+               hx-get="{% url 'process_manager:process_table' %}"
+               hx-trigger="keyup delay:500ms, changed, search, every 1s"
+               hx-target="div.table-container"
+               hx-swap="outerHTML">
+        <div id="table-container"
+             hx-get="{% url 'process_manager:process_table' %}"
              hx-swap="outerHTML"
              hx-trigger="load"></div>
       </form>
```

Comment:
Unsure how much it really matters, but:

```suggestion
        <input class="form-control join-item"
               type="search"
               name="search"
               placeholder="Search..."
               hx-get="{% url 'process_manager:process_table' %}"
               hx-trigger="input changed delay:500ms, search, every 1s"
               hx-target="div.table-container"
               hx-swap="innerHTML">
        <div id="table-container"
             hx-get="{% url 'process_manager:process_table' %}"
             hx-swap="innerHTML"
             hx-trigger="load"></div>
      </form>
```
where:

1. `hx-trigger="input changed delay:500ms, ..."` also handles mouse copy/paste, which `keyup` does not
2. `innerHTML` as we only need to refresh the inner contents now -- see other template comment